ddtrace/internal/telemetry/writer.py

```diff
@@ -104,7 +108,7 @@ def periodic(self):
         if integrations:
             self._app_integrations_changed_event(integrations)
 
-        telemetry_requests = self._flush_events_queue()
+        telemetry_requests = self.flush_events_queue()
```

Comment:
I made `flush_events_queue` "public" to use it in a test. It seems like a helpful function to expose. I could revert this change since it's unreleased to this fix.